### PR TITLE
docs: remove out of date GDEs

### PR DESCRIPTION
**Pull request by StephenFluin**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit

## PR Type
What kind of change does this PR introduce?
- [x] Documentation content changes

### CODE REVIEW

**Comment by gabrielferreirapro** [comment]
Hey @StephenFluin , aren't this guys GDE anymore?

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -349,14 +349,6 @@
     "bio": "Sander is a versed developer with over 4 decades of practice under his belt. He is also an Google Developer Expert for web, specializing in Angular. Organizer of meetups and conferences. Helping out others wherever he can. When he is not breathing code, he is fiddling around with IOT, photography, science and anything that might vaguely is gadget-like! Thinks he a master of the grill, but in reality you probably don't get a food-poisoning ;) Also, and actually the most important thing to him, he is a father of 4, and has the most patient girlfriend in the universe.",
     "groups": ["GDE"]
   },
-  "filipbech": {
-    "name": "Filip Bruun Bech-Larsen",
```
Comment (gabrielferreirapro) [inline aio/content/marketing/contributors.json:353]: I searched for this guy at https://developers.google.com/community/experts/directory and I found him.

**Review by gabrielbussolo — CHANGES_REQUESTED** [review]
Hey @StephenFluin, I searched for this guys at "Directory of Experts". And I found this guy Filip Bruun Bech-Larsen, can u verify and submit again?
Thanks :)

File: aio/content/marketing/contributors.json (hunk 1 above)
Comment (StephenFluin) [reply aio/content/marketing/contributors.json:353]: He's still a Web Technologies GDE, just not an Angular one.

Comment (gabrielferreirapro) [reply aio/content/marketing/contributors.json:353]: I'm sorry, I hadn't noticed the difference.

**Review by gabrielbussolo — APPROVED** [review]
I checked the people that were removed.
Looks good for me :+1:

**Review by gkalpak — CHANGES_REQUESTED** [review]
Please remove the corresponding images from [aio/content/images/bios/], if they are no longer needed 🙏 

[1]: https://github.com/angular/angular/tree/9056e480739523f28ffd6dfca652d09c1bc6bb67/aio/content/images/bios

**Review by gkalpak — APPROVED** [review]

**Review by IgorMinar — APPROVED** [review]

**Pull request closed without merging**